Automattic/wp-calypso
# Fix/etk fse universal theme clean up

#### Proposed Changes

Clean up for ETK Full site editing activation code.
Removed unused/unnecessary code related to activating FSE and kept code to hide "Menus" submenu in Calypso.

The logic for hiding the `Menus` submenu was kept to ensure consistency due to the fact that it was hidden in the FSE beta.

This should be merged after the D84305-code is merged.

#### Testing Instructions

* Activate FSE enabled theme (Blank Canvas). You should not see the `Menus` submenu under the appearance tab.
* Activate a non-FSE enabled theme (Dara). You should see the `Menus` submenu under the appearance tab.
* The AMP menu should still appear in calypso/wp-admin side menu if plugin is activated.
* Ensure site-editor is still loading correctly.

#### Pre-merge Checklist

Complete applicable items on this checklist **before** merging into trunk. Inapplicable items can be left unchecked.

Both the PR author and reviewer are responsible for ensuring the checklist is completed.


Related to #61107
